src/styles/card.scss

```
@@ -0,0 +1,155 @@
+.card {
+  box-sizing: border-box;
+  margin: 50px 0 46px var(--card-margin-left);
+  max-height: 408px;
+  width: var(--card-max-width);
+  background-color: #fff;
+  border: var(--card-border) solid #f3f3f3;
+  border-radius: var(--card-border-radius);
+}
+
+.card__sectionimage {
```

Get used to use all the advantages of SASS
```suggestion
.card {
  box-sizing: border-box;
  margin: 50px 0 46px var(--card-margin-left);
  max-height: 408px;
  width: var(--card-max-width);
  background-color: #fff;
  border: var(--card-border) solid #f3f3f3;
  border-radius: var(--card-border-radius);
  
  &__section-image {
    ...
  }
}
```